src/renderers/0-3.js

```diff
@@ -50,7 +51,12 @@ export default class Renderer {
   }
 
   renderMarkupSection ([type, TagName, markers], nodeKey) {
-    const element = <TagName key={nodeKey}>{[]}</TagName>;
+    let props;
+    if (TagName in this.sections) {
+      props = this.sections[TagName];
```

Comment:
So "sections" here are not real sections, rather than additional props for the markup tags!

Reply:
Going through this PR https://github.com/dailybeast/thedailybeast/pull/1626/files I understand now what are you trying to achieve: pass inline styles to the markups.
I think that passing styles to the mobiledoc is architecturally not a good approach. Styles must be separated from the JavaScript code, otherwise it makes it harder to support and override.
Our mobiledoc-renderer supports rendering markup components (I did this before for the links in text paragraphs).

My opinion is that we should create Headline and Subheadline React components with their own styles and render them as markups in mobiledoc.

Reply:
Yeah, ok I see that. @dfairaizl Do you approve that approach?

Reply:
@oliviadangelo Better talk to him in person